src/main/scala/bank/Controller.scala

```diff
@@ -0,0 +1,43 @@
+package bank
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.server.Directives._
+import spray.json._
+
+class Controller extends DefaultJsonProtocol with SprayJsonSupport {
+
+  var accounts = List[Account](
+    Account(
+      accountNumber = "123",
+      currentBalance = 9999,
+      currency = "PHP",
+      customerFirstName = "Lloyd",
+      customerLastName = "Edano"
+    )
+  )
+
+  def createAccount = post {
+    path("api" / "accounts" ) {
+      entity(as[Account]) { account=>
+        accounts = accounts :+ account
+        complete(Created, "Account created".toJson )
+      }
+    }
+  }
+
+  def getAccount = get {
```

Comment:
- put type annotation for public methods
- change name to getAccountByAccNum
- getAccountByAccNum is not a scope of this task, make a separate task/PR